src/Turbo/src/Broadcaster/TwigBroadcaster.php

```diff
@@ -21,20 +21,15 @@
  */
 final class TwigBroadcaster implements BroadcasterInterface
 {
-    private $broadcaster;
-    private $twig;
-    private $templatePrefixes;
-    private $idAccessor;
-
     /**
      * @param array<string, string> $templatePrefixes
      */
-    public function __construct(BroadcasterInterface $broadcaster, Environment $twig, array $templatePrefixes = [], ?IdAccessor $idAccessor = null)
-    {
-        $this->broadcaster = $broadcaster;
-        $this->twig = $twig;
-        $this->templatePrefixes = $templatePrefixes;
-        $this->idAccessor = $idAccessor ?? new IdAccessor();
+    public function __construct(
+        private BroadcasterInterface $broadcaster,
+        private Environment $twig,
+        private array $templatePrefixes = [],
+        private IdAccessor $idAccessor = new IdAccessor(),
```

Comment:
This is a BC break, the argument is no longer nullable.

Reply:
Absolutely, thank you. fixed